setup.cfg

```diff
@@ -24,6 +24,10 @@ install_requires =
   google-auth-oauthlib
   google-api-python-client
   requests
+  selenium==4.3.0
+  pandas==1.4.3
+  undetected_chromedriver
+  webdriver-manager==3.8.3
```

Comment:
not an expert on this but seems like there is a way to make it optional and by default not included? https://stackoverflow.com/questions/6237946/optional-dependencies-in-distutils-pip

Reply:
done!